config/candidates-api.yml

```diff
@@ -0,0 +1,111 @@
+---
+openapi: 3.0.0
+info:
+  version: v1
+  title: Apply candidates API
+  contact:
+    name: DfE
+    email: becomingateacher@digital.education.gov.uk
+  description: |
+    API for candidates from DfE’s Apply for teacher training service.
+servers:
+- description: Sandbox (test environment)
+  url: https://sandbox.apply-for-teacher-training.service.gov.uk/candidates-api
+- description: Production
+  url: https://www.apply-for-teacher-training.service.gov.uk/candidates-api
+paths:
+  "/candidates":
+    get:
+      summary: Get a list of candidates
+      description: This endpoint returns a list of candidates
+      responses:
+        '200':
+          description: A list of candidates
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Candidates"
+  "/candidates/{updated_since}":
+    get:
+      summary: Get a list of candidates updated since
+      parameters:
+        - in: path
+          name: updated_since
+          schema:
+            type: string
+            format: date-time
+            example: 2021-05-20T12:34:00Z
+          required: true
+          description: candidate data updated since
+      responses:
+        '200':
+          description: Returns a list of candidates created or updated since a certain date and time
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/CandidateList"
+
+components:
+  schemas:
+    Candidates:
```

Comment:
Is this a dupe of the below `CandidateList`?

Reply:
Yeah, pretty much. It was following the pattern seen in the [vendor api](https://github.com/DFE-Digital/apply-for-teacher-training/blob/ea767f7d2ee75097b66d48c39fdcd503bdea64fc/config/vendor-api-v1.yml#L365) where `MultipleApplicationsResponse` and `SingleApplicationResponse` both reference "#/components/schemas/Application"